executor/common.h

```diff
@@ -467,8 +469,13 @@ static uintptr_t syz_extract_tcp_res(uintptr_t a0, uintptr_t a1, uintptr_t a2)
 	}
 
 	struct tcp_resources* res = (struct tcp_resources*)a0;
+#if defined(SYZ_EXECUTOR) || defined(SYZ_HANDLE_SEGV)
```

Comment:
I didn't find a way to do this via preprocessor.

Simply matching `NONFAILING(.*);\n` wouldn't work, sine some kvm code does:
```
NONFAILING(
  do_access_1();
  do_access_2();
);
```

I could either write a simple parser to parse such uses of `NONFAILING` correctly or change kvm code to use one `NONFAILING` per line. Which approach is better?